test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -62,12 +62,20 @@ call_log_manager::process_call_log()
 void
 call_log_manager::process_call_log_entry(json call_log_entry)
 {
-    const std::string method_name = call_log_entry["MethodName"].get<std::string>();
+    const std::string method_name = call_log_entry["method_name"].get<std::string>();
+    std::cout << "Processing entry " << method_name << std::endl;
```

Comment:
Printing information doesn't seem helpful, especially if there are thousands of call log entries? @bvpvamsikrishna

Reply:
Sorry, I had this for debugging, I'll remove it.

Reply:
Fixed.